packages/core/src/_common/base/login.js

```diff
@@ -18,7 +18,7 @@ const Login = (() => {
     };
 
     const redirectToSignUp = () => {
-        window.open('https://deriv.com?action=signup');
+        window.open('https://deriv.com/signup/');
```

Comment:
@arash-fs there is also a mention of the same link in trader files. should we update that too @mahdipourziaei-binary ?

Reply:
@negarn @arash-fs if the function `redirectToSignUp` is being used in Trader, then yes.